### DISCUSSION TITLE
Remove unnecessary // ok comments from UnnecessaryParenthesesCheck.java

### DISCUSSION THREAD

**Discussion by ghost** [checkstyle/checkstyle#15806] [General]
I am a begineer and i have removed the unnecessary "// ok " comments from src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java
and i have even generated  the PR but like @romani  shared me  https://checkstyle.org/beginning_development.html#Starting_Development 
so what should i do next ?

**Comment by romani** [comment]
all guidance will be at https://github.com/checkstyle/checkstyle/pull/15803
